.github/workflows/pull-request-labeler.yml

```diff
@@ -0,0 +1,23 @@
+name: "Label PRs"
```

Comment:
suggestion: can we rename this to pr-auto-labeling to be more consistent with the other check names?

Reply:
Updated to suggested name!